applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -155,6 +155,15 @@ Button 1:
     * If pressed for more than three seconds, it initiates the factory reset of the device.
       Releasing the button within a 3-second window of the initiation cancels the factory reset procedure.
 
+LED 2:
+   If the :kconfig:option:`CONFIG_BRIDGED_DEVICE_BT` kconfig option is set to ``y`` shows the current state of Bluetooth LE Connectivity Manager state.
```

Comment:
Could we not mention BLE Connectivity Manager name here? I think it's quite technically detailed information that is not relevant for UI description. I would suggest just calling it e.g. "current state of bridge's Bluetooth LE connectivity".